### PR TITLE
Login

### PR DESCRIPTION
**Pull request by amitshethe**
Devise gem installed

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -11,6 +11,12 @@
   </head>
 
   <body>
+  <% if notice %>
+  <p class="alert alert-success"><%= notice %></p>
```
Comment (nyadav-deqode) [inline app/views/layouts/application.html.erb]: @amitshethe Indentations seems incorrect

File: test/models/user_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
```
Comment (nyadav-deqode) [inline test/models/user_test.rb]: @amitshethe This file can be deleted

**Review by nyadav-deqode — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (amitshethe) [reply app/views/layouts/application.html.erb]: corrected them in controller setup branch

File: test/models/user_test.rb (hunk 2 above)
Comment (amitshethe) [reply test/models/user_test.rb]: corrected them in controller setup branch

**Pull request merged**